pkg/generator/generator.go

```diff
@@ -129,26 +133,25 @@ func (g Generator) GenerateRawConfig(ctx context.Context, installation, app stri
 	}
 
 	// 5.
-	secretsContext, err := g.getWithPatchIfExists(
+	secretContext, err := g.getWithPatchIfExists(
 		ctx,
-		"installations/"+installation+"/secrets.yaml",
+		"installations/"+installation+"/secret.yaml",
 		"",
 	)
 	if err != nil {
 		return "", "", microerror.Mask(err)
 	}
 
-	// 6.
 	if g.decryptTraverser != nil {
```

Comment:
Why would it ever be nil? I missed that in the previous review. If this is for tests then we can create a very simple traverser returning the same values.

Reply:
Good point. I just added a `DecryptTraverser` to test file anyways, so let's do it.